### PR TITLE
Update INSTALLATION.md

### PR DESCRIPTION
**Pull request by DhruvKadam-git**
1. Added a note about using venv for virtual environments.

2. Updated the Node.js version requirement to the latest LTS  version.

3. Added a section for installing specific versions of Taipy.

4. Clarified the build process for JavaScript bundles.

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -22,9 +22,24 @@ If you are running in a virtual environment, you will have to issue the command:
...
+```bash
+python -m venv myenv
+source myenv/bin/activate  # On Windows use `myenv\Scripts\activate`
+pip install taipy
+```
+
 These commands install the `taipy` package in the Python environment with all its
 dependencies.
 
+## Installing a Specific Version
```
Comment (trgiangdo) [inline INSTALLATION.md]:
```suggestion
## Installing a Specific Version from PyPI
```

File: INSTALLATION.md [hunk 2]
```diff
@@ -22,9 +22,24 @@ If you are running in a virtual environment, you will have to issue the command:
...
+
+To install a specific version of Taipy, use the following command:
+```bash
+pip install taipy==<version>
+```
+Replace `<version>` with the desired version number.
```
Comment (trgiangdo) [inline INSTALLATION.md]:
```suggestion
Replace `<version>` with a specific version number of Taipy.
The list of all released Taipy versions can be found [here](https://pypi.org/project/taipy/#history).
```

**Review by FlorianJacta — COMMENTED** [review]
I agree with @trgiangdo remarks

**Comment by DhruvKadam-git** [comment]
I have commited changes which was suggested by @trgiangdo please review it.

**Review by FlorianJacta — APPROVED** [review]
LGTM

**Comment by DhruvKadam-git** [comment]
If its ok pls review it other reviewers

**Comment by DhruvKadam-git** [comment]
Please review my PR

**Comment by DhruvKadam-git** [comment]
Please review my PR

**Review by jrobinAV — APPROVED** [review]

**Review by FabienLelaquais — APPROVED** [review]
Looks good!

**Comment by jrobinAV** [comment]
Thank you for your contribution!

**Pull request merged**